src/pages/index.tsx

```diff
@@ -1,45 +1,49 @@
 import { trpc } from "@/utils/trpc";
+import React from "react";
+import Link from "next/link";
+import { DxDeck } from "@prisma/client";
+
 export default function Home() {
-  const { data, isLoading } = trpc.useQuery(["get-random-card"]);
+  const { data, isLoading } = trpc.useQuery(["get-decks"]);
 
-  if (isLoading)
-    return <div className="text-black flex justify-center">Loading...</div>;
-  if (data)
-    return (
-      <div className="h-screen w-screen flex justify-center items-center">
-        <div className="h-5/6 w-5/6 relative flex justify-center items-center">
-          <div className="absolute text-orangeweboxfordblue-primary text-5xl shadow-2xl top-1">
-            Source Material
-          </div>
-          <div className="absolute bg-orangeweboxfordblue-tertiary border-orangeweboxfordblue-border border-4 shadow-2xl text-4xl top-1 right-1">
+  type DeckProps = {
+    decks: DxDeck[];
+  };
+  type DeckState = {};
+
+  class Decks extends React.Component<DeckProps, DeckState> {
```

Comment:
Would it make sense to separate this out into a component file? Thinking down the road for a landing / login page, or if there ends up being more logic needed on the index page it would flow easier.

Reply:
Yes! I think a lot of these pages should be broken up into helper pages and more extensible representations. For now, though, I think we can run with the 1-page set-up and talk about structure next meeting. Down to change the way we handle things now, though, if that makes sense!
@nleech16 @tliebert Thoughts?